cfme/tests/services/test_generic_service_catalogs.py

```diff
@@ -82,7 +82,15 @@ def test_service_generic_catalog_bundle(catalog_item):
     cells = {'Description': row_description}
     row, __ = wait_for(requests.wait_for_request, [cells, True],
         fail_func=requests.reload, num_sec=900, delay=20)
-    assert row.last_message.text == 'Request complete'
+    # Success message differs between 5.6 and 5.7
+    success_message = 'Service '
+    if version.current_version() >= '5.7':
+        # 5.7 success message includes catalog item name in brackets
+        success_message += '[{}] '.format(bundle_name)
+        success_message += 'Provisioned Successfully'
```

Comment:
I think this line should have one less indentation, so that only `'[{}] '.format(bundle_name)` part of the message is dependent on 5.7.

Believe the intention is to create two messages:
* `'Service Provisioned Successfully'` for 5.6
* `'Service [{}] Provisioned Successfully'.format(bundle_name)` for 5.7